health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -812,23 +812,24 @@ export const UICustomizations = {
               customName: campaignName
             });
           };
 
           const onActionSelect = (e) => {
-            if (e.name == "MP_ACTIONS_EDIT_SETUP") {
-              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${1}&microplanId=${row.id}&campaignId=${
-                row.campaignDetails.id
-              }`;
-            }
+            if (e.name == "MP_ACTIONS_EDIT_SETUP") { 
+                window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${String(parseInt(row?.additionalDetails?.key || '0') + 1)}&microplanId=${row.id}&campaignId=${
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)